input/CIB_Lenz.yml

```diff
@@ -0,0 +1,17 @@
+mapper_class: MapperCIBLenz
```

Comment:
We actually have 3 different frequencies for this map, so we should create 3 different files. Have a look at these ones I created for Baptiste's project:
```
/mnt/extraspace/damonge/Datasets/CIB_Lenz/map_353_2.5_20gp.yml
/mnt/extraspace/damonge/Datasets/CIB_Lenz/map_545_2.5_20gp.yml
/mnt/extraspace/damonge/Datasets/CIB_Lenz/map_857_2.5_20gp.yml
```

Reply:
Generated three files now!